src/server/services/brp.ts

```diff
@@ -80,6 +70,13 @@ export function transformBRPNotifications(data: BRPData, compareDate: Date) {
     }
   );
 
+  const documentsExpiringDuringPeak = data.identiteitsbewijzen?.filter(
+    (document) => {
+      const afloop = new Date(document.datumAfloop)
+      return ID_BEWIJS_PEAK_DATE_START <afloop && new Date(document.datumAfloop) < ID_BEWIJS_PEAK_DATE_END 
```

Comment:
Als je hem zo schrijft is het was leesbaarder

```suggestion
      return afloop >= ID_BEWIJS_PEAK_DATE_START && afloop <= ID_BEWIJS_PEAK_DATE_END 
```